### ISSUE TITLE
[Bug] Unable to dynamically update pinnedBottomRowData in AG Grid Vue 3 (v32.0.0)

### ISSUE THREAD

**Issue by XAJ-Code** [ceolter/ag-grid#9222]
Title: [BUG] Unable to update pinnedBottomRowData in AG Grid Vue 3 (v32.0.0)

Description:
I'm trying to update the `pinnedBottomRowData` in AG Grid Vue 3 (version 32.0.0), but it's not working as expected.

Current code:
`<template>
<AgGridVue
:pinnedBottomRowData="pinnedBottomRowData"
...other props
/>
</template>
<script setup>
const gridApi = shallowRef()
const pinnedBottomRowData = ref([])
// Trying to update pinned bottom row
const updateData = async () => {
const bottomData = {...} // some data
pinnedBottomRowData.value = [bottomData]
gridApi.value?.setGridOption('pinnedBottomRowData', [bottomData])
}
</script>`
Expected behavior:
- The pinned bottom row should update when the data changes

Actual behavior:
- The pinned bottom row is not updating when using the above methods

Environment:
- AG Grid Version: 32.0.0
- Vue Version: 3.x
- Browser: [Your browser]

Question:
What is the correct way to update pinnedBottomRowData in AG Grid Vue 3 (v32.0.0)?

**Comment by JasonTheProgrammer** [comment]
Do your grid options include a callback for `getRowId`? It might be that you're running afoul of the bug reported in #8319 . That was identified as fixed in a later version, but as I've just found, it's still present in the current latest version v32.3.3 of the TS / JS library per #9358 , so your mileage with a later version of the Vue version might vary.

A workaround you might apply is a call to `gridApi.refreshCells()` after setting the pinned row data. I found that worked for me.
